daemon/mgr/container.go

```diff
@@ -393,6 +396,38 @@ func (mgr *ContainerManager) Stop(ctx context.Context, name string, timeout time
 	return nil
 }
 
+// Pause pauses a running container.
+func (mgr *ContainerManager) Pause(ctx context.Context, name string) error {
+	var (
+		err error
+		c   *Container
+	)
+
+	if c, err = mgr.container(name); err != nil {
+		return err
+	}
+
+	c.Lock()
+	defer c.Unlock()
+
+	if c.meta.Config == nil || c.meta.ContainerState == nil {
+		err := fmt.Errorf("no container found by %s", name)
+		return httputils.NewHTTPError(err, http.StatusNotFound)
```

Comment:
Please rebase to master, since there is a commit https://github.com/alibaba/pouch/pull/270 merged now and this status code part is no longer correct.
Please use the correct way to make it. Thanks a lot.